<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportCertDlg</class>
 <widget class="QDialog" name="ImportCertDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Import certificate</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout_4">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Slot name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="mSlotNameText">
       <property name="palette">
        <palette>
         <active>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>221</red>
             <green>221</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>221</red>
             <green>221</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled/>
        </palette>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item row="0" column="6">
      <widget class="QLineEdit" name="mLoginText">
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="palette">
        <palette>
         <active>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>221</red>
             <green>221</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>221</red>
             <green>221</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>240</red>
             <green>240</green>
             <blue>240</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="mSessionText">
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="palette">
        <palette>
         <active>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>221</red>
             <green>221</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>221</red>
             <green>221</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>240</red>
             <green>240</green>
             <blue>240</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Session</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="mSlotIDText">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="palette">
        <palette>
         <active>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>221</red>
             <green>221</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>221</red>
             <green>221</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>240</red>
             <green>240</green>
             <blue>240</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Slot ID</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_5">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item row="0" column="1">
      <widget class="QCheckBox" name="mSubjectInCertCheck">
       <property name="text">
        <string>subject in cert</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="4" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Cert Path</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QCheckBox" name="mUseSKICheck">
       <property name="text">
        <string>Use SKI</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="mCertPathText"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="mSubjectText"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLineEdit" name="mLabelText"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Label</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="mSubjectLabel">
       <property name="text">
        <string>Subject</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="mIDText"/>
     </item>
     <item row="3" column="2">
      <widget class="QComboBox" name="mSubjectTypeCombo"/>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="mFindBtn">
       <property name="text">
        <string>Find</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>PubKeyInfo</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="mPubKeyInfoText"/>
     </item>
     <item row="2" column="2">
      <widget class="QCheckBox" name="mUseSPKICheck">
       <property name="text">
        <string>Use SPKI</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Attributes</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="1">
      <widget class="QComboBox" name="mPrivateCombo"/>
     </item>
     <item row="0" column="0">
      <widget class="QCheckBox" name="mPrivateCheck">
       <property name="text">
        <string notr="true">Private</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QCheckBox" name="mModifiableCheck">
       <property name="text">
        <string notr="true">Modifiable</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QComboBox" name="mModifiableCombo"/>
     </item>
     <item row="0" column="4">
      <widget class="QCheckBox" name="mTokenCheck">
       <property name="text">
        <string notr="true">Token</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QComboBox" name="mTokenCombo"/>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="mCopyableCheck">
       <property name="text">
        <string>Copyable</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="mCopyableCombo"/>
     </item>
     <item row="1" column="2">
      <widget class="QCheckBox" name="mDestroyableCheck">
       <property name="text">
        <string>Destroyable</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QComboBox" name="mDestroyableCombo"/>
     </item>
     <item row="1" column="4">
      <widget class="QCheckBox" name="mTrustedCheck">
       <property name="text">
        <string>Trusted</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QComboBox" name="mTrustedCombo"/>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="mStartDateCheck">
       <property name="text">
        <string>StartDate</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDateEdit" name="mStartDateEdit"/>
     </item>
     <item row="2" column="2">
      <widget class="QCheckBox" name="mEndDateCheck">
       <property name="text">
        <string>EndDate</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QDateEdit" name="mEndDateEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="Line" name="line_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>mSlotNameText</tabstop>
  <tabstop>mSlotIDText</tabstop>
  <tabstop>mSessionText</tabstop>
  <tabstop>mLoginText</tabstop>
  <tabstop>mSubjectInCertCheck</tabstop>
  <tabstop>mLabelText</tabstop>
  <tabstop>mIDText</tabstop>
  <tabstop>mUseSKICheck</tabstop>
  <tabstop>mPubKeyInfoText</tabstop>
  <tabstop>mUseSPKICheck</tabstop>
  <tabstop>mSubjectText</tabstop>
  <tabstop>mSubjectTypeCombo</tabstop>
  <tabstop>mCertPathText</tabstop>
  <tabstop>mFindBtn</tabstop>
  <tabstop>mPrivateCheck</tabstop>
  <tabstop>mPrivateCombo</tabstop>
  <tabstop>mModifiableCheck</tabstop>
  <tabstop>mModifiableCombo</tabstop>
  <tabstop>mTokenCheck</tabstop>
  <tabstop>mTokenCombo</tabstop>
  <tabstop>mCopyableCheck</tabstop>
  <tabstop>mCopyableCombo</tabstop>
  <tabstop>mDestroyableCheck</tabstop>
  <tabstop>mDestroyableCombo</tabstop>
  <tabstop>mTrustedCheck</tabstop>
  <tabstop>mTrustedCombo</tabstop>
  <tabstop>mStartDateCheck</tabstop>
  <tabstop>mStartDateEdit</tabstop>
  <tabstop>mEndDateCheck</tabstop>
  <tabstop>mEndDateEdit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ImportCertDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ImportCertDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
